src/components/ToggleSwitch/index.tsx

```diff
@@ -11,6 +11,7 @@ import {
   TouchableOpacityProps
 } from 'react-native';
 import dimen from '../../utils/dimen';
+import {normalizeFontSize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `normalizeFontSize` from the `fonts` utility module is a good addition. It helps to maintain consistency in font sizes across different screen sizes.
```diff
+ import {normalizeFontSize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->